apps/daimo-web/src/app/profile/[...addr]/route.tsx

```diff
@@ -14,6 +14,9 @@ const pfpCache = new LRUCache<string, Blob>({
   max: 1000,
 });
 
+// Opt out of caching for all data requests in the route segment
+export const dynamic = "force-dynamic";
```

Comment:
@koredefashokun I think this may be one of the causes of #879 -- on my local sim I could reproduce connecting a new account for an already loaded profile and getting a cached `getEthereumAccount` result by nextJS

Reply:
Ah. Interesting. Does this fix the issue for you locally? We could also look at this if it doesn't: https://nextjs.org/docs/app/building-your-application/caching#fetch-optionscache